generators/app/templates/webapp/Component.ts

```diff
@@ -121,8 +95,9 @@ export default class Component extends UIComponent {
 		return new Promise((resolve, reject) => {
 			const FCL = ((this.getRootControl() as View).byId('fcl') as FlexibleColumnLayout);
 			if (!FCL) {
-				(this.getRootControl() as View).attachAfterInit((oEvent: UI5Event) => {
-					resolve(((oEvent.getSource() as View).byId('fcl') as FlexibleColumnLayout));
+				(this.getRootControl() as View).attachAfterInit((oEvent: UI5Event) => {<% if (gte11160) { %>
```

Comment:
@petermuessig , can you validate this is correct? not sure this is from 1.116 and above.

Reply:
Changed by using the newly provided event types.

Reply:
Should be correct now